projects/vendor/src/yotpo/baseyotpo/baseyotpo.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable, ElementRef } from '@angular/core';
```

Comment:
Please name the parent folder service so we can add more services to it eventually.

Reply:
done